vhdbuilder/packer/install-dependencies.sh

```diff
@@ -268,53 +267,38 @@ done
 # v1.20.2
 # v1.20.5
 # NOTE that we keep multiple files per k8s patch version as kubeproxy version is decided by CCP.
-PATCHED_HYPERKUBE_IMAGES="
+KUBE_PROXY_IMAGE_VERSIONS="
 1.17.13
+1.17.13-hotfix.20210310.1
 1.17.16
+1.17.16-hotfix.20210310.1
 1.18.8-hotfix.20200924
+1.18.8-hotfix.20201112.1
 1.18.10-hotfix.20210118
+1.18.10-hotfix.20210310.1
 1.18.14-hotfix.20210118
 1.18.14-hotfix.20210322
 1.18.17-hotfix.20210322
 1.19.1-hotfix.20200923
+1.19.1-hotfix.20200923.1
 1.19.3
 1.19.6-hotfix.20210118
```

Comment:
most of the added versions are no longer supported, why do we add them back?

Reply:
to make old clusters happier.
will remove the legacy images after we bumped the rp with the toggle changes.